css/src/components/chip.css

```diff
@@ -137,12 +137,37 @@
   border-radius: 10px;
 }
 
+.Chip-icon--right:hover {
+  background-color: var(--secondary);
+  border-radius: 10px;
+}
+
+.Chip-icon--right:active {
+  background-color: var(--secondary-dark);
+  border-radius: 10px;
+}
```

Comment:
Please give `border-radius: 10px` css to `Chip-icon--right` class directly instead of using the same css in different condition to remove redundancy